translations/translations.json

```diff
@@ -3125,7 +3125,6 @@
       "negativeRating": "ይህ ገጽ ጠቃሚ አይደለም",
       "positiveRating": "ይህ ገጽ ጠቃሚ ነው",
       "contactMailAddress": "ለጥያቄዎች ኢሜይል",
-      "optionalInfo": "አማራጭ /ሊተው የሚችል/",
```

Comment:
Why did you delete this one?

Reply:
Because it was just a leftover from a previous merge, it was moved to `optional` in common.